applicationinsights/applicationinsights-js.d.ts

```diff
@@ -0,0 +1,488 @@
+// Type definitions for ApplicationInsights-JS v0.22.14
+// Project: https://github.com/Microsoft/ApplicationInsights-JS
```

Comment:
This is correct. We have different SDKs for Node.js and Javascript: 
* https://github.com/Microsoft/ApplicationInsights-JS/ 
* https://github.com/Microsoft/ApplicationInsights-node.js/